quic-client/tests/quic_client.rs

```diff
@@ -151,4 +157,140 @@ mod tests {
         exit.store(true, Ordering::Relaxed);
         t.await.unwrap();
     }
+
+    #[test]
+    fn test_quic_bi_direction() {
```

Comment:
Is the point of this test just to test that we can get the returned endpoint from the receiver, and both send stuff and receive stuff using the same endpoint?